docker-compose.yaml

```diff
@@ -25,9 +25,13 @@ services:
       AZURE_STORAGE_ACCOUNT_NAME: ${AZURE_STORAGE_ACCOUNT_NAME:-devstoreaccount1}
       AZURE_STORAGE_USE_CONNECTION_STRING: "true"
       NOTIFY_API_KEY: ${NOTIFY_API_KEY}
+      NOTIFY_API_KEY_LETTER: ${NOTIFY_API_KEY_LETTER}
       NOTIFY_EMAIL_TEMPLATE_KEY: ${NOTIFY_EMAIL_TEMPLATE_KEY}
       DOC_NOTIFY_API_KEY: ${NOTIFY_API_KEY}
+      DOC_NOTIFY_API_KEY_LETTER: ${NOTIFY_API_KEY_LETTER}
       DOC_NOTIFY_EMAIL_TEMPLATE_KEY: ${NOTIFY_EMAIL_TEMPLATE_KEY}
+      STATEMENT_RECEIVER_API_VERSION: v1
+      STATEMENT_RECEIVER_ENDPOINT: http://host.docker.internal:3022
```

Comment:
does this not need setting according to env variables for our deployments to snd and up?

Reply:
I know we set in the helm config map but I have a feeling you may also need to set it at docker compose to be on the safe side. I'm a little fuzzy on this though.

Reply:
I'm not sure about this either; I have compared with doc generator and we have the value in exactly the same places.

Reply:
Lets go with it then